opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -1,10 +1,18 @@
 import abc
 import logging
+import re
 import types as python_types
 import typing
+from collections import OrderedDict
 
 from opentelemetry.trace.status import Status
 from opentelemetry.util import types
+from opentelemetry.util.tracestate import (
+    _DELIMITER_PATTERN,
+    _MEMBER_PATTERN,
+    _TRACECONTEXT_MAXIMUM_TRACESTATE_KEYS,
+    _is_valid_pair,
```

Comment:
These imports shouldn't be named with underscore prefixes right?

Reply:
I think these are for internal use and should have leading underscore. Ideally users should only rely on `TraceState` and `inject/extract` API.